server/tserver/src/main/java/org/apache/accumulo/tserver/TabletServer.java

```diff
@@ -3002,6 +3009,24 @@ public TServerInstance getTabletSession() {
     }
   }
 
+  private void checkWalCanSync(ServerContext context) {
+    VolumeChooserEnvironment chooserEnv =
+        new VolumeChooserEnvironmentImpl(VolumeChooserEnvironment.ChooserScope.LOGGER, context);
+    String logPath = fs.choose(chooserEnv, ServerConstants.getBaseUris(context)) + Path.SEPARATOR
```

Comment:
Its possible there are multiple volumes that could be chosen for logging.  Not sure what the best way to get them all is though.

Reply:
Oh I see now that the check is done again later for each log file. So it will eventually catch it.